### PR TITLE
Integrate with Knowtify webhook

### PR DESCRIPTION
**Pull request by DaneLyons**

### CODE REVIEW

**Comment by harrietgrace** [comment]
Hey @DaneLyons, thanks for your PR! 

Our old template style is obviously a little verbose for your needs,  so if you could do a quick style refactor [along these lines](https://github.com/segmentio/integration-iron-io/blob/master/lib/index.js), that would be great :) That example is using the mapper to restructure their payload a little, but if you don't need it you can just remove that file and the reference to it.

**Comment by DaneLyons** [comment]
Thanks for looking into the PR @harrietgrace!

I cleaned up some comments and added https to our endpoint. I tried removing the mapper and references to it but tests were throwing errors so I left it in.

File: test/index.js [hunk 1]
```diff
@@ -53,23 +55,29 @@ describe('Knowtify', function(){
       });
     });
 
+    /*
     describe('page', function(){
       it('should map basic page', function(){
         test.maps('page-basic');
       });
     });
+    */
```
Comment (harrietgrace) [inline test/index.js]: Nitpick: we can remove all these commented out ones :)

**Comment by harrietgrace** [comment]
Nice! Thanks for getting into that so quickly. 
Unless you have future plans to do different things with the track and identify calls, we could even further remove duplication by having a helper function send the requests (as they're currently identical) like this: 

```
Knowtify.prototype.identify = request;
Knowtify.prototype.track = request;

function request(payload, fn){
  return this
    .post()
    .query({ token: this.settings.apiToken })
    .type('json')
    .send(payload)
    .end(this.handle(fn));
};
```

**Comment by DaneLyons** [comment]
@harrietgrace thanks for the help! I just took your suggestions and cleaned things up a bit.

**Comment by amillet89** [comment]
looks great, thanks for the PR!

**Pull request merged**